test/unit/pusherConstructorTest.php

```diff
@@ -78,22 +78,53 @@ public function testEncryptedOptionWillBeOverwrittenByHostAndPortOptionsSetHostA
 				'port' => '3000'
 			);
 			$pusher = new Pusher( 'app_key', 'app_secret', 'app_id', $options );
-			
+
 			$settings = $pusher->getSettings();
 			$this->assertEquals( 'http', $settings[ 'scheme' ] );
 			$this->assertEquals( $options[ 'host' ], $settings[ 'host' ] );
 			$this->assertEquals( $options[ 'port' ], $settings[ 'port' ] );
-		}	
-		
+		}
+
 		public function testSchemeIsStrippedAndIgnoredFromHostInOptions() {
 			$options = array(
 				'host' => 'https://test.com'
 			);
 			$pusher = new Pusher( 'app_key', 'app_secret', 'app_id', $options );
-			
+
 			$settings = $pusher->getSettings();
 			$this->assertEquals( 'http', $settings[ 'scheme' ] );
 			$this->assertEquals( 'test.com', $settings[ 'host' ] );
-		}		
+		}
+
+    public function testClusterSetsANewHost() {
```

Comment:
This project has a lot of trailing white spaces. My editor is wired to
automatically remove them. That is why this PR is so "busy". If needs be I
can make another PR ?!

On Wed, Feb 17, 2016 at 4:39 PM, Mike Pye <notifications@github.com> wrote:

> In test/unit/pusherConstructorTest.php
> <https://github.com/pusher/pusher-http-php/pull/91#discussion_r53192016>:
>
> >  			$settings = $pusher->getSettings();
> >  			$this->assertEquals( 'http', $settings[ 'scheme' ] );
> >  			$this->assertEquals( 'test.com', $settings[ 'host' ] );
> > -		}		
> > +		}
> > +
> > +    public function testClusterSetsANewHost() {
>
> The project appears to use tabs exclusively, but this PR adds a lot of
> spaces.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pusher/pusher-http-php/pull/91/files#r53192016>.
>



-- 
Alexandru Topliceanu